competition/competition.go

```diff
@@ -0,0 +1,95 @@
+package competition
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+
+	"strconv"
+
+	"github.com/project-regista/scout/configuration"
+	"github.com/project-regista/scout/scout4neo"
+)
+
+type Competition struct {
+	Data []struct {
+		ID      int    `json:"id"`
+		Cup     bool   `json:"cup"`
+		Name    string `json:"name"`
+		Active  bool   `json:"active"`
+		Country struct {
+			ID      int    `json:"id"`
+			Name    string `json:"name"`
+			IsoCode string `json:"iso_code"`
+			Flag    string `json:"flag"`
+		} `json:"country"`
+	} `json:"data"`
+}
+
+func requestCompetitions() Competition {
```

Comment:
`func requestCompetition() (Competition, error)`

to enjoy multiple returns values and handle all the errors in the main package's function. This way we always know where is the first place to go when the code breaks and we can handle all the possible errors in the same place. Also, is easier to test, since we dont have to check the stderr in every function